.NET/Microsoft.Recognizers.Text.Number/Parsers/BaseCJKNumberParser.cs

```diff
@@ -112,7 +112,7 @@ public override ParseResult Parse(ExtractResult extResult)
             }
 
             // Japanese determine the subtype
-            if (Config.CultureInfo.Name == "ja-JP" && ret != null)
+            if ((Config.CultureInfo.Name == "ja-JP" || Config.CultureInfo.Name == "ko-KR") && ret != null)
```

Comment:
Please add a `@TODO` comment here to refactor this check. We'll plan for it to happen soon.

Reply:
Sure